checker/src/org/checkerframework/checker/units/UnitsTools.java

```diff
@@ -27,21 +29,31 @@
 
 // TODO: add fromTo methods for all useful unit combinations.
 public class UnitsTools {
```

Comment:
The static final fields are now sorted alphabetically by category to make it easier to maintain in the future.